loader/include/Geode/utils/Task.hpp

```diff
@@ -882,7 +948,6 @@ namespace geode {
             return ListenerResult::Propagate;
         }
 
-        // todo: i believe alk wanted tasks to be in their own pool

```

Comment:
dont remove the comment

Reply:
but wouldn't a todo for something that's implemented be kinda useless...?

Reply:
its not implemented.

Reply:
it will be once my [question](https://github.com/geode-sdk/geode/pull/1175#discussion_r1854919162) is answered

Reply:
i have an implementation i just need you to answer the question i asked like 4 hours ago

Reply:
At the moment your PR has the same implementation as we have currently, so please keep the comment in.

Reply:
comment is back